init.c

```diff
@@ -4275,7 +4220,7 @@ void mutt_init(int skip_sys_rc, struct ListHead *commands)
   /* Do we have a locale definition? */
   if (((p = getenv("LC_ALL")) != NULL && p[0]) || ((p = getenv("LANG")) != NULL && p[0]) ||
       ((p = getenv("LC_CTYPE")) != NULL && p[0]))
-    set_option(OPT_LOCALES);
+    OPT_LOCALES = true;
```

Comment:
Why this change?

Reply:
`OPT_LOCALES` is a global variable, which is bundled with the general neomutt config bools.
(packed into the `Options` variable).

`OPT_LOCALES` really belongs with the multi-byte code, so I've split it out into a real bool global variable.